rocketmq-connect/rocketmq-connect-runtime/src/main/java/org/apache/rocketmq/connect/runtime/service/PositionManagementServiceImpl.java

```diff
@@ -154,7 +178,6 @@ public void onCompletion(Throwable error, String key, Map<ByteBuffer, ByteBuffer
             boolean changed = false;
             switch (PositionChangeEnum.valueOf(key)) {
                 case ONLINE_KEY:
-                    mergePositionInfo(result);
```

Comment:
How to synchronize data when starting ？

Reply:
The offset of the starting node generally lags behind that of the existing node. If the node is restarted after being stopped for a period of time, direct merging may overwrite the latest offset, resulting in data duplication. 
   In addition, the offset of the starting node has been synchronized to the corresponding topic, and other nodes have actually obtained all the offsets, so I think it is not necessary to directly merge the offsets of the starting node.